tests/test_plottingfuncs.py

```diff
@@ -58,13 +80,14 @@ def test_plot_and_save_bounding_box(
 
 @pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
 def test_plot_and_save_zrange(minicircle_grain_gaussian_filter: Grains, plotting_config: dict, tmpdir) -> None:
-    """Test plotting with colorbar"""
+    """Tests plotting of the zrange scaled image"""
     plotting_config["zrange"] = [-10, 10]
     plotting_config["core_set"] = True
     fig, _ = plot_and_save(
         minicircle_grain_gaussian_filter.images["gaussian_filtered"],
-         tmpdir, "08_5-z_threshold.png",
-         title="Raw Height", 
+         Path(tmpdir),
```

Comment:
Perhaps pedantic but I've been going through and putting the explicit arguments into all the tests to help avoid confusion and so readers don't have to remember the positional arguments.

```suggestion
         data=minicircle_grain_gaussian_filter.images["gaussian_filtered"],
         output_dir=Path(tmpdir),
         filename="08_5-z_threshold.png",
```

Mainly because my memory isn't great and having had a break I was having to look up the options.

Reply:
@ns-rse I'll keep this in mind and make the changes!

I've also seen that pathlib can use a full path variable called `tmp_path` instead of the python LocalPath `tmpdir`. This has been an issue in a few of my tests where using the LocalPath with pathlib doesn't work hence I've used `Path(tmpdir)`. Do you want me to make the switch to make all the tests neater?

Reply:
I would say that using the full path variable would be better, as I believe there are issues with cross-compatibility between different and more niche installation environments (eg: in personal projects, I find that using local paths breaks on my Windows 10 system but functions fine on my MacOS system)